src/hooks/v3/V3ProjectState.ts

```diff
@@ -0,0 +1 @@
+export { useV2ProjectState as useV3ProjectState } from '../v2/V2ProjectState'
```

Comment:
Reuse V2 hook in V3. If it turns out we need a custom V3 hook, all we need to do is remove this line, define a useV3ProjectState fn and export that instead.